### DISCUSSION TITLE
What does this log mean (Flux Dev) ? :  "unknown tensor..."

### DISCUSSION THREAD

**Discussion by olivbrau** [leejet/stable-diffusion.cpp#521] [Q&A] [answered]
Hi,
here is my command : 
`sd --diffusion-model .\flux_dev\flux1-dev-q4_0.gguf --clip_l .\flux_dev\clip_l.safetensors --vae .\flux_dev\ae.safetensors --t5xxl .\flux_dev\t5xxl_fp16.safetensors --sampling-method euler --steps 30 --cfg-scale 1.0 -H 768 -W 512 -t 20  -p "..."`

In the logs, I get this message :
> unknown tensor 'text_encoders.t5xxl.transformer.encoder.embed_tokens.weight | f16 | 2 [4096, 32128, 1, 1, 1]' in model file

Does it mean I have something wrong in the model file ?
In fact, the processing continue after that and seems OK.

Thanks for your help

Olivier

**Comment by stduhpf** [comment, marked as answer]
No, there's nothing wrong with your file. It just means that the t5 model file includes a tensor ("encoder.embed_tokens.weight") that isn't used by sdcpp. I belive it's a copy of the tensor "shared.weight", which is used instead.
Maybe this tensor should be explicitely ignored to avoid this log.
